scripts/dev/update_version.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2018 Andy Mender <andymenderunix@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+from datetime import date
+import subprocess
+
+from lxml import etree
+
+# TODO: move to global constants?
+appdata_path = "misc/qutebrowser.appdata.xml"
+version_xpath = '//*[@type="desktop"]/releases'
+
+
+def bump_version(version_leap = "patch"):
+    """Update qutebrowser release version in .bumpversion.cfg
+
+    :param version_leap: define the jump between versions ("major", "minor", "patch")
```

Comment:
Please use the google syntax for docstrings (see [this part](https://github.com/qutebrowser/qutebrowser/blob/master/doc/contributing.asciidoc#style-conventions) of the contribution docs; same for other docstrings as well).